cli_tools/osconfig_agent/google-osconfig-agent.goospec

```diff
@@ -9,7 +9,7 @@
     "osconfig_agent.exe": "<ProgramFiles>/Google/OSConfig/osconfig_agent.exe"
   },
   "releaseNotes": [
-    "0.1.0 - Initial packaging"
```

Comment:
nit: we don't want to keep this in the list and "backfill" the 0.2.0 notes?

Reply:
Eh, I'm not sure how much I even want to keep here, we could just not have any notes until later. Once we go beta Ill keep each entry, but up to then many entries are mostly useless to end users and after beta you really don't care about what was done pre-release